src/test/java/com/epam/tc/hw1/DivideTest.java

```diff
@@ -0,0 +1,17 @@
+package com.epam.tc.hw1;
+
+import com.epam.tat.module4.Calculator;
+import org.assertj.core.api.AssertionsForClassTypes;
+import org.testng.annotations.Test;
+
+public class DivideTest {
+
+    @Test(groups = {"divmult"},
+            dataProviderClass = DataProviderForCalculator.class,
+            dataProvider = "division data")
+
+    public void divisionTest(int a, int b, int result) {
+        Calculator calculator = new Calculator();
```

Comment:
May be `calculator` could be extracted to the field in `Base` class for these tests. Initialisation could be done in `@BeforeMethod` for this field.
All test classes could be extended from the created base class